schedule/yast/yast2_gui/yast2_gui_sle.yaml

```diff
@@ -36,6 +36,7 @@ conditional_schedule:
                 - yast2_gui/yast2_lan_restart_vlan
                 - yast2_gui/yast2_lan_restart_bond
                 - yast2_gui/yast2_storage_ng
+                - console/yast2_nfs_server
```

Comment:
ncurses test in this test suite seems not be suitable.

Reply:
you are right. removed